src/app/popups/add-family-member/add-family-member.component.html

```diff
@@ -0,0 +1,60 @@
+<md-dialog-content>
+  <div class="family-member-popup">
+    <div (click)="close()" class="family-member-close">
+      X
+    </div>
+    <table class="family-member-form">
+      <tr>
+        <td><b>First Name</b></td>
+        <td>
+          <input 
+            type="text" 
+            name="first-name"
+            mdInput
+            [(ngModel)]="familyMemberFirstName">
+        </td>
+      </tr>
+      
+      <tr>
+        <td><b>Last Name</b></td>
+        <td>
+          <input 
+            type="text" 
+            name="last-name"
+            mdInput
+            [(ngModel)]="familyMemberLastName">  
+        </td>
+      </tr>
+  
+      
+      <tr>
+        <td><b>Subject ID</b></td>
```

Comment:
can you add `*` and somewhere on the popup that
`* only number to add to family ID needed` something like that! I will ask PM for better text